src/storages/inRedis/ImpressionCountsCacheInRedis.ts

```diff
@@ -0,0 +1,38 @@
+import { Redis } from 'ioredis';
+import { ImpressionCountsCacheInMemory } from '../inMemory/ImpressionCountsCacheInMemory';
+import { IImpressionCountsCacheInRedis } from '../types';
+
+const IMPRESSION_COUNT_REFRESH_RATE = 300000; // 300.000 ms = start after 5 mins
+
+export class ImpressionCountsCacheInRedis extends ImpressionCountsCacheInMemory implements IImpressionCountsCacheInRedis {
+
+  private readonly key: string;
+  private readonly redis: Redis;
+  private handle: any;
+  
+  constructor(key: string, redis: Redis, impressionCountsCacheSize?: number) {
+    super(impressionCountsCacheSize);
+    this.key = key;
+    this.redis = redis;
+    this.onFullQueue = () => { this.postImpressionCountsInRedis(); };
+  }
+  
+  postImpressionCountsInRedis(){
+    const counts = this.pop();
+    const keys = Object.keys(counts);
+    const pipeline = this.redis.pipeline();
+    keys.forEach(key => {
+      pipeline.hincrby(this.key, key, counts[key]);
+    });
+    return pipeline.exec();
```

Comment:
Important: need to handle key expiration (3600 seconds) similar to what we do in ImpressionsCacheInRedis::track.
And also handle `exec` rejected promises. We should log an error if the operation fail, something similar to what we do in `EventsCacheInRedis::track`